### PR TITLE
Fixed formatting, added "Contributing" section

### PR DESCRIPTION
**Pull request by drphilmarshall**
While reviewing https://github.com/lsst-dm/nb_lsst_io/pull/1 I suggested to @jonathansick that the notebook aspect _docs_ needed notes on how to contribute, and then realized that its also this repo that we might get good contributions to. So, I have enhanced the README!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+> **Note:** there is a special branch called `prod` in this repo that is continuously deployed into production. Do not update this branch unless you know what you are doing. (Better instructions that "know what you are doing" will be forthcoming once the workflow settles down).
 
 ## General setup
-
-Once you have started your jupyterlab session, start a terminal and
-
+Once you have started your JupyterLab session, start a terminal and
+```bash
 cd ~/notebooks
 git clone https://github.com/lsst-sqre/notebook-demo.git
```
Comment (jonathansick) [inline README.md]: The notebook-demo repo is automatically cloned into your `~/notebooks` dir, so this won't be necessary for the science platform.

**Comment by SimonKrughoff** [comment]
Unless there is a reason not to do so, we are clearing outputs in committed notebooks.  The workflow I've been working with is that PRs are merged by repository owners when approved and additions are cherry picked to prod by the merger.

**Comment by SimonKrughoff** [comment]
Saying a bit more, I think Phil's plan (private communication) to generate static HTML in CI is a good one.  That means there's really no downside to clear outputs.

**Comment by jonathansick** [comment]
OK neat. If the outputs are saved I can render the notebooks in the nb.lsst.io docs too.

**Comment by athornton** [comment]
I'm going to close this; the notebook repository and thus the instructions have changed in the meantime, and I think it's probably no longer relevant.

**Pull request closed without merging**